### PR TITLE
Android New IA Redirects

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**

### CODE REVIEW

File: config/redirects [hunk 1]
```diff
@@ -15,35 +15,38 @@ raw: realm/graphql/graphql-types-and-resolvers/index.html -> ${base}/graphql/typ
...
-raw: realm/dotnet/quick-start -> ${base}/sdk/dotnet/
-raw: realm/dotnet/reads -> ${base}/sdk/dotnet/
-raw: realm/dotnet/realms -> ${base}/sdk/dotnet/
-raw: realm/dotnet/reference -> ${base}/sdk/dotnet/
-raw: realm/dotnet/relationships -> ${base}/sdk/dotnet/
-raw: realm/dotnet/sync-data -> ${base}/sdk/dotnet/
-raw: realm/dotnet/work-with-multiple-users -> ${base}/sdk/dotnet/
-raw: realm/dotnet/writes -> ${base}/sdk/dotnet/
\ No newline at end of file
+raw: android/access-custom-user-data -> ${base}/sdk/android/advanced-guides/custom-user-data
```
Comment (cbush) [inline config/redirects:18]: Think these need to start with
```suggestion
raw: realm/android/access-custom-user-data -> ${base}/sdk/android/advanced-guides/custom-user-data
```

**Comment by cbush** [comment]
Don't forget to add realm/android/ -> realm/sdk/android/

**Comment by nathan-contino-mongo** [comment]
@cbush Fixed, can you take another look?

**Review by cbush — APPROVED** [review]
Seems correct

File: config/redirects [hunk 2]
```diff
@@ -15,35 +15,38 @@ raw: realm/graphql/graphql-types-and-resolvers/index.html -> ${base}/graphql/typ
...
+raw: android/livedata -> ${base}/sdk/android/livedata
+raw: android/manage-email-password-users -> ${base}/sdk/android/examples/email-password-users
+raw: android/migrate -> ${base}/sdk/android/migrate
+raw: android/migrations -> ${base}/sdk/android/advanced-guides/migrations
+raw: android/mongodb-realm -> ${base}/sdk/android
+raw: android/mongodb -> ${base}/sdk/android/examples/mongodb-remote-access
+raw: android/notifications -> ${base}/sdk/android/examples/react-to-changes
+raw: android/objects -> ${base}/sdk/android/fundamentals/object-models-and-schemas
+raw: android/open-a-realm -> ${base}/sdk/android/examples/open-and-close-a-local-realm
+raw: android/query-engine -> ${base}/sdk/android/fundamentals/live-queries
```
Comment (nlarew) [inline config/redirects]: Did we port this to a usage example? Sounds like it's **Filter Data** in iOS

Comment (nathan-contino-mongo) [reply config/redirects]: There is no filter data page in Android, live queries is the equivalent.

**Pull request merged**